pkg/java/pom/parse.go

```diff
@@ -245,7 +245,7 @@ func (p *parser) analyze(pom *pom, exclusions map[string]struct{}) (analysisResu
 
 	// Extract and merge dependencies under "dependencyManagement"
 	depManagement := p.dependencyManagement(pom.content.DependencyManagement.Dependencies.Dependency, props)
-	depManagement = p.mergeDependencyManagement(parent.dependencyManagement, depManagement)
+	depManagement = p.mergeDependencyManagements(rootDependencyManagement, depManagement, parent.dependencyManagement)
```

Comment:
Can you please add a comment about the precedence order?

Reply:
Done